integrations/langchain/src/databricks_langchain/genie.py

```diff
@@ -64,6 +101,8 @@ def GenieAgent(
     include_context: bool = False,
     message_processor: Optional[Callable] = None,
     client: Optional["WorkspaceClient"] = None,
+    truncate_results: bool = False,
```

Comment:
Is there a reason to reintroduce this `truncate_results` argument? I believe @aravind-segu removed this since it's now deprecated functionality

Reply:
The reason we are trying to deprecate this flag btw is that it's generally better to truncate results in client-side code, not in this API